views/game.html.erb

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" class="mdl-js">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="assets/favicon.ico" rel="icon">
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
+    <link href="assets/css/styles.css" rel="stylesheet">
+    <script src="assets/js/jquery.min.js"></script>
+    <script src="assets/js/bootstrap.min.js"></script>
+    <title>Codebreaker Web</title>
+  <body>
+    <section id="introduction">
+      <div class="container">
+        <div class="row fill-viewport align-items-center justify-content-center">
+          <div class="col-12 col-md-6">
+            <div class="row">
+              <div class="col-md-auto">
+                <h1 class="text-white title">Codebreaker 2018</h1>
+              </div>
+            </div>
+            <p class="lead text-white-70">
+              Hello, test!
+              Try to guess 4-digit number, that consists of numbers in a range from 1 to 6.
```

Comment:
I suggest moving static text to constants

Reply:
i would agree with you if it wasn't the only occurrence of this text. Else i will have to move to constants a lot of stuff.